examples/AdvancedOperations/AddAppCampaign.php

```diff
@@ -0,0 +1,466 @@
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Creates the language campaign criteria.
+        $languageIds = [
+            1000, // English
+            1003  // Spanish
+        ];
+        foreach ($languageIds as $languageId) {
+            // Creates a campaign criterion.
+            $campaignCriterion = new CampaignCriterion([
+                'campaign' => new StringValue(['value' => $campaignResourceName]),
+                'type' => CriterionType::LANGUAGE,
+                'language' => new LanguageInfo([
+                    'language_constant' => new StringValue([
+                        'value' => ResourceNames::forLanguageConstant($languageId)
+                    ])
+                ])
+            ]);
+
+            // Creates a campaign criterion operation.
+            $campaignCriterionOperation = new CampaignCriterionOperation();
+            $campaignCriterionOperation->setCreate($campaignCriterion);
+
+            $campaignCriterionOperations[] = $campaignCriterionOperation;
+        }
+
+        // Submits the criteria operations and prints their information.
+        $campaignCriterionServiceClient = $googleAdsClient->getCampaignCriterionServiceClient();
+        $response = $campaignCriterionServiceClient->mutateCampaignCriteria(
+            $customerId,
+            $campaignCriterionOperations
+        );
+
+        printf(
+            "Created %d campaign criteria with resource names:%s",
+            $response->getResults()->count(),
+            PHP_EOL
+        );
+
+        foreach ($response->getResults() as $createdCampaignCriterion) {
+            /** @var CampaignCriterion $createdCampaignCriterion */
+            printf("\t%s%s", $createdCampaignCriterion->getResourceName(), PHP_EOL);
+        }
+    }
+
+    /**
+     * Creates an ad group for a given campaign.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $campaignResourceName the resource name of the campaign to add the ad group to
+     * @return string the resource name of the newly created ad group
+     */
+    private static function createAdGroup(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        // Creates an ad group.
+        // Note that the ad group type must not be set.
+        // Since the advertising_channel_sub_type is APP_CAMPAIGN,
+        //   1. you cannot override bid settings at the ad group level.
+        //   2. you cannot add ad group criteria.
+        $adGroup = new AdGroup([
+            'name' => new StringValue(['value' => 'Earth to Mars cruises ' . uniqid()]),
+            'status' => AdGroupStatus::ENABLED,
```

Comment:
Should this be PAUSED to conform to other examples? We usually set campaign and ad group status to PAUSED as a best practice.

Reply:
Note that the entire campaign is paused. This is based on the previous AdWords example that had a paused campaign and an enabled Ad Group.

Reply:
I checked and we have many occurrences of ENABLED ad groups used in Shopping and Hotel code examples. I do not have strong feelings about this one as the campaign is disabled.

@AnashOommen is there any directive that needs to be followed on this?